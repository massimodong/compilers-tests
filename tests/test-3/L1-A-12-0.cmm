q5jhSVcXWnnnvxXnkeJo {
}sVoNwpt6B814u ( int _6Fwu29Tg1Fkj7qdTck , struct 
  lj6CMmLM c1ZdsEi ) {
  float aUC3kQx [ 1166828533 ] = PgsLlom || HkFltWg 
    && kOYYlvb < Usxmwwh3vYvmdijpzZ - ! Aiu53F84k_DPz_ 
    ( CmJIFCdXPU = GSM5Jtmioe21NEbLe <= Yz7XZ8uqMi49BKzX 
    [ eN_zRejfKTst7vx && a9Od_be6P2E != rFlNK0hcEt1OXRIw5p 
    - ! ( U7C9ZIk = X6b || MReHi >= baDbkJqQ2Bf2_doSR 
    + kn_ ( VUGbuD9j6X , vWkGXmdaBg4FuBOXaX > ( K8I ( 
    fitPcDYAKJ = _b7Xaf == HKd3zYjcyFVo ) %) - 116818233 
    ) ) + 928567113.1869300815 ] . rqZf28hIqVbN1b ) / 
    L2ERJRekYX0oQtbRt ( ) * - ! CuZ0XoGuUBPbvzVFqN ;
}
struct {
}IyqcOUJetk2 
;
struct xj3OUnK K6NpIN9NjlH1fZQfIT 
;
int Ss5EPr2uKcO78LS9msTe , muM 

;
struct {
};
int ;
int vCeRlRnc0Y5_m3 ( ) {
  if ( TwrutZqepcfvU )   yytexerboT ;
else   if ( Og3mo_7oH2IOM ) {
    int i4DgCvG5nlsBF , zinOZHoDZVFuzL8 ;
    if ( yA_ )     Xh9iqpV ;
else     while ( ZejrZn )     vSJxgVm9xMR ;


    if ( ErG0tV3txqmz )     if ( GsL3QnhtTakwXgU ) 
          v04bIOTbjr ;
else     if ( hHjkG9okT4f1cgrtlYN     if ( SG8Y ) 
          return R0pLwAgGj0J ;




'  }

}
struct N3_ThjVr9T0p0MzPZ2F ;
