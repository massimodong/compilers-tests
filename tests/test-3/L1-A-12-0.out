Error Type B at Line 1
Error Type A at Line 10
Error Type B at Line 35
Error Type A at Line 41
